sources/packages/backend/apps/api/src/services/application-offering-change-request/application-offering-change-request.service.ts

```diff
@@ -156,15 +156,18 @@ export class ApplicationOfferingChangeRequestService {
 
   /**
    * Gets application offering request change list.
-   * @param locationId location id.
-   * @param paginationOptions options to execute the pagination.
    * @param statuses list of status that need to be included in the query.
+   * @param paginationOptions options to execute the pagination.
+   * @param options method options:
+   * - `locationId`: location for authorization.
    * @returns list of requested application offering changes.
    */
   async getSummaryByStatus(
-    locationId: number,
-    paginationOptions: PaginationOptions,
     statuses: ApplicationOfferingChangeRequestStatus[],
+    paginationOptions: PaginationOptions,
+    options?: {
```

Comment:
IMO I believe, as the locationId is just a single optional field, it would be better to have it as an optional parameter rather having it in options, in the future if we have additional parameters, we can extend it.

Reply:
That's how I agreed to do when talking to Andrew. Using the options object is the way we are doing all across the other parts of the application so I think that's ok to be like that.